src/components/Auth/SignUpForm.js

```diff
@@ -36,102 +32,106 @@ const SignUpForm = ({ toggleActiveForm }) => {
         setErrorMessage(Object.values(err.response.data).join(''));
       });
   };
+
   if (isLoggedIn) {
     return <Redirect to={referer} />;
   }
+
+  if (auth && auth.authTokens) {
+    return <p>Welcome!</p>;
+  }
+
   return (
-    <>
-      {(auth && auth.authTokens) || isLoggedIn ? (
-        <p>Welcome!</p>
-      ) : (
-        <>
+    <Box
+      component={Form}
+      display="flex"
+      flexWrap="wrap"
+      noValidate
+      autoComplete="off"
+      onSubmit={onSubmit}
+      data-testid="signupForm"
+      validationResolver={validationResolver}
+      defaultValues={defaultValue}
+    >
+      <Box component="h1" fontSize={18}>
+        Create an account
+      </Box>
+
+      <Field
+        as={TextField}
+        fullWidth
+        variant="outlined"
+        margin="dense"
+        name="firstName"
+        label="First Name"
+        id="first-name"
```

Comment:
is there a reason `id` and `name` are different?

Reply:
@sebbel No strong reason. `name` is associated with the schema validation which is tied to form data where it make sense to use camelCase. We can update id as same as name. It's just dashed case is popular across css classes which is why i followed the same convention over ids.